website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 1
+title: OpenShift Local
+description: Creating an OpenShift Local instance
+keywords: [podman desktop, podman, containers, pods, migrating, kubernetes, openshift]
+tags: [migrating-to-kubernetes, openshift]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating an OpenShift Local instance
+
+[Red Hat OpenShift Local](https://developers.redhat.com/products/openshift-local/overview) manages a minimal OpenShift or MicroShift cluster on your workstation for local development and testing.
+
+With Podman Desktop and the OpenShift Local extension, you can manage your OpenShift Local instances.
+
+#### Prerequisites
+
+- [Register a Red Hat account](https://developers.redhat.com/register).
+
+#### Procedure
+
+1. Install the _OpenShift Local_ extension: on to **Dashboard**, click **OpenShift Local <icon icon="fa-solid fa-download" size="lg" />**.
+1. Install the OpenShift Local binaries, when on the **Dashboard**, you see _Podman Desktop was not able to find an installation of OpenShift Local_.
+
+   <Tabs groupId="operating-systems">
+   <TabItem value="win" label="Windows">
+
+   1. In the **OpenShift Local** tile, click **<icon icon="fa-solid fa-rocket" size="lg" /> Install**.
+   1. When prerequisites are missing, follow the instructions.
+   1. In the **Red Hat OpenShift Local** screen, click **Yes** to start the installation.
+   1. Follow the installation program instructions.
+   1. Reboot to finalize system changes.
+
+   </TabItem>
+   <TabItem value="mac" label="macOS">
+
+   1. In the **OpenShift Local** tile, click **<icon icon="fa-solid fa-rocket" size="lg" /> Install**.
+   1. When prerequisites are missing, follow the instructions.
+   1. In the **Red Hat OpenShift Local** screen, click **Yes** to start the installation.
+   1. Follow the installation program instructions.
+   1. Reboot to finalize system changes.
+
+   </TabItem>
+   <TabItem value="linux" label="Linux">
```

Comment:
Don't we support installation on Linux through the extension ?

Reply:
No (confirmed in chat by @gbraad)

Reply:
We don't have any installer for Linux.